pkg/compare/testdata/ReferenceV2InlineCapturegroups/localinvalidCapturegroupsLateDetectionout.golden

```diff
@@ -1,4 +1,4 @@
-Skipping during Input contains additional files from supported file extensions (json/yaml) that do not contain a valid resource, error: :18 capturegroup contains spaces or linebreaks.
+Skipping during Input contains additional files from supported file extensions (json/yaml) that do not contain a valid resource, error: :]`.
```

Comment:
This looks weird because of the TODO on line 691 of compare.go... not going to solve it today though.